### ISSUE TITLE
Players can't break or place blocks in the nether

### ISSUE THREAD

**Issue by iVillager** [GeyserMC/Geyser#2552]
### Describe the bug

The issue happens only with the nether world. The nether world is vanilla like, there's nothing blocking the normal vanilla mechanics. Tested with end world and overworld too.
Currently running last Geyser build on my Proxy. 

### To Reproduce

1. Go to the nether
2. Try to break something with no items, with pickaxes or pales
3. Try to place something

### Expected behaviour

The expected behaviour is that you can place or break blocks.. 

### Server Version and Plugins

Server version is:
This server is running Airplane-Purpur version git-Airplane-6 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

### Geyser Dump

https://dump.geysermc.org/DMHbHHWxN3NKKSJray3ZfJNaUUPQGjwz

### Geyser Version

Geyser version 1.4.3-SNAPSHOT (git-master-9a 87959) (Java: 1.17.1, Bedrock: 1.17.10/1.17.11 - 1.17.30)

### Minecraft: Bedrock Edition Version

Bedrock: 1.17.10/1.17.11 - 1.17.30

**Comment by lukeeey** [comment]
is this an issue on a paper server

**Comment by iVillager** [comment]
Yes it's a paper server running purpur

**Comment by Chew** [comment]
Then it's not Paper, it's Purpur... 🤨 

Run "vanilla" Paper found here: https://papermc.io/downloads

**Comment by Camotoy** [comment]
Do note that we have another instance of this issue that is running (an outdated version of) Paper: https://dump.geysermc.org/M5gB43oT3YmFOOeETKBAJfftnKdgXvWb

**Comment by iVillager** [comment]
> Then it's not Paper, it's Purpur... raised_eyebrow
> 
> Run "vanilla" Paper found here: https://papermc.io/downloads

I wrote that the server version was:

This server is running Airplane-Purpur version git-Airplane-6 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

**Comment by iVillager** [comment]
> Do note that we have another instance of this issue that is running (an outdated version of) Paper: https://dump.geysermc.org/M5gB43oT3YmFOOeETKBAJfftnKdgXvWb

I don't think that the version of paper would afflict how geyser works in our server

**Comment by Chew** [comment]
If you're not going to do what we've requested, at the very least update your version of Airplane, which is currently outdated: https://ci.tivy.ca/job/Airplane-1.16/11/

**Comment by DeltaX86** [comment]
Hello,
I have the same problem, is there a solution?
Airplane and geyser last version.
best regards

**Comment by Chew** [comment]
Please provide a dump, and see if the issue persists on Paper.

**Comment by Camotoy** [comment]
I cannot replicate the issue yet - could someone upload a world file I can test with? Assuming it occurs with no plugins.

**Comment by iVillager** [comment]
I updated my server version to the latest Airplane 1.16
The issue still happens
